### ISSUE TITLE
Examples in SimulationData API reference is showing Simulation instead of SimulationData

### ISSUE THREAD

**Issue by tomflexcompute** [flexcompute-readthedocs/tidy3d-docs#183]
The API reference on SimulationData contains a few examples such as using the classmethod:
<img width="632" alt="image" src="https://user-images.githubusercontent.com/116006359/233239264-0def146b-849a-491d-860e-caa4e3beda68.png">

The example code is using Simulation instead of SimulationData. The classmethod is not used correctly in some cases either, such as when talking about `from_hdf5` the example is calling `to_hdf5`.

**Comment by tylerflex** [comment]
Added a fix for this in the front end.

The example should use `from_hdf5`, I fixed that.

However, we use `Simulation` because it's defined at the `Tidy3dBaseModel` base class, so all tidy3d objects inherit this method and the same docstring. So `Simulation` was just chosen as an example for how to use the method. there's no way (that I know of) to try to have it update to the inherited class so I think it's fine.
